docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: "3.8"
+services:
+  server:
+    image: klakegg/hugo:0.83.1
+    command: server
```

Comment:
Would `--navigateToChanged` (the best hugo-feature that is not enabled by default) work when changing source locally?

Reply:
I didn't even get it to auto update without restarting the container. Probably because of some issues with the WSL and windows file system not working great together. I don't plan on writing lots of docs, so it doesn't matter for me, but I'm much more confident running unknown software in docker than native, so I think this is a useful addition anyway.

Reply:
Ok, then I suggest doing this in a seperate PR, then we can test & improve the docker experience without delaying the other link fixes,

Reply:
Ok, I'll keep my docker-compose locally for now.

Reply:
Added an issue https://github.com/Altinn/altinn-studio-docs/issues/285